.gitignore

```diff
@@ -3,6 +3,7 @@
 *.o
 *.so
 *.swp
+*.yardoc
```

Comment:
What paths are you expected by this pattern?

I think that `/.yardoc/` is correct entry.